build/testConfig.js

```diff
@@ -1,12 +1,17 @@
 const assign = require('object-assign');
 const nodePath = require('path');
+const os = require('os');
```

Comment:
hi, is this file change needed for this pr ?

if so can you motivate it in the pr description?

Reply:
done

> There is an addition on the testConfig related to the output folder to allow to correctly include png images imported directly in js files, such as the marker icon. This change is needed to successfully test the MarkeUtils that generate icons used inside the new test of this PR